app/assets/javascripts/ajax_mixpanel_events.js

```diff
@@ -0,0 +1,35 @@
+var ajaxMixpanelEvents = (function() {
+  var init = function() {
+    $("[data-track-click]").on("click", function (e, options = {}) {
+      if (options.trackedEvent === true) {
+        return;
+      }
+
+      e.preventDefault();
+      e.stopPropagation();
+
+      var clickedElement = $(e.target);
+      var eventData = {
+        event_name: "clicked_link",
+        controller_action: window.mixpanelData.controller_action,
+        full_path: window.mixpanelData.full_path,
+        data: {
+          link_text: clickedElement.text().trim(),
+        }
```

Comment:
It doesn't look link this is adding "idme" to the event data or the event name. The event name submitted should be "click_idme_signin"